src/background.js

```diff
@@ -4,8 +4,20 @@
 
 //block message js core requests
 chrome.webRequest.onBeforeRequest.addListener(
-  function (details) {
-    return {cancel: true};
-  },
-  { urls: ["https://*.ses.nsw.gov.au/js/messages/create?v=*"] },
-  ["blocking"]);
+	function (details) {
```

Comment:
wow this function is amazing. So amazing I think it deserves a comment explaining why you have to do these dirty dirty hacks.